pennylane_lightning/core/src/simulators/lightning_qubit/algorithms/AdjointJacobianLQubit.hpp

```diff
@@ -273,6 +276,8 @@ class AdjointJacobian final
                                 (*H_lambda_storage)[ind].size());
                 H_lambda->push_back(sv);
             }
+        } else {
+            PL_ABORT("Undefined memory storage location for StateVectorT.");
```

Comment:
The drop in coverage is probably associated with this addition. We might need to consider it in the tests.

Reply:
done